src/build/optimize-pipe.ts

```diff
@@ -10,9 +10,51 @@
 
 import * as stream from 'stream';
 
+import * as gulpif from 'gulp-if';
```

Comment:
ah, no. I meant exteral vs internal to project. basically separating './'  and '../' imports